parsl/addresses.py

```diff
@@ -156,3 +157,21 @@ def get_any_address() -> str:
     if addr == '':
         raise Exception('Cannot find address of the local machine.')
     return addr
+
+
+def tcp_url(address: str, port: Union[str, int, None] = None) -> str:
+    """Construct a tcp url safe for IPv4 and IPv6"""
+    stripped_address = address.strip('[]')
```

Comment:
this code is cruft  already maybe? - IP addresses don't have `[]` in or around them

Reply:
Fair enough, I'll remove that.